packages/augur-tools/package.json

```diff
@@ -18,8 +18,8 @@
     "docker:push": "bash ./docker-builder/push.sh",
     "docker:build-and-push": "yarn docker:build && yarn docker:push",
     "docker:pull": "./docker-builder/pull.sh",
-    "docker:geth": "docker network create augur 2>&1 >/dev/null; docker run --rm -e GETH_VERBOSITY=4 -e PERIOD_TIME=1 -it -p 8545:8545 -p 8546:8546 --name geth --network augur augurproject/dev-node-geth:v1.9.9",
-    "docker:geth:detached": "docker network create augur 2>&1 >/dev/null; docker run --detach --rm -e GETH_VERBOSITY=4 -e PERIOD_TIME=1 -it -p 8545:8545 -p 8546:8546 --name geth --network augur augurproject/dev-node-geth:v1.9.9",
+    "docker:geth": "docker network create augur-tools_default 2>&1 >/dev/null; docker run --rm -e GETH_VERBOSITY=4 -e PERIOD_TIME=1 -it -p 8545:8545 -p 8546:8546 --name geth --network augur-tools_default augurproject/dev-node-geth:v1.9.9",
```

Comment:
Do we need to do this still?

Reply:
AFAIK the network name change is still needed.